lib/qrolo.dart

```diff
@@ -27,526 +28,277 @@ import 'package:qrolo/src/html/media/utilities/is_media_device_camera_available.
 import 'package:qrolo/src/jsqr.dart' show jsQR;
 
 import 'package:js/js.dart' show JS, anonymous;
+import 'package:qrolo/src/media.dart';
 
 const int defaultScanIntervalMilliseconds = 500;
 
 /// The QRolo scanner widget
+/// !IMPORTANT: This widget needs to be bound in a sized box or other container
+/// Other Flutter throws unbound render flex hit test errors
 class QRolo extends StatefulWidget {
+  /// clickToCapture to show a button to capture a Data URL for the image
+  final bool clickToCapture;
+
+  const QRolo({this.clickToCapture = false, Key? key}) : super(key: key);
+
   @override
   _QRoloState createState() => _QRoloState();
 
   static const MethodChannel _channel = MethodChannel('qrolo');
 
   static Future<String?> get platformVersion async {
     final version = await _channel.invokeMethod<String>('getPlatformVersion');
-
-    return version;
   }
 
-  /// Whether to show the camera button overlaid on scanner video stream view
```

Comment:
Temporary replacement

Will vanish from history when root issue is found in debugging

Harder to debug due to interop